wp-content/plugins/core/src/Routes/Routes_Definer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Defines all regular and REST routes to register.
+ */
+declare( strict_types=1 );
```

Comment:
These will later be moved to `<?php declare( strict_types=1 );` but not required for you to change that now since so many files will have to be fixed anyways.